modules/tensor_mechanics/doc/content/source/kernels/StressDivergenceTensors.md

```diff
@@ -16,6 +16,26 @@ _use_displaced_mesh_ in mechanics problems and the TensorMechanics Master Action
 
 !include modules/tensor_mechanics/common/supplementalStressDivergenceKernels.md
 
+## Use with Planar Models
+
+When used with 2D planar models (plane stres, plane strain, or generalized plane strain), 
+it is used to compute the residuals for the in-plane response. In all of these cases,
+it assumed that the out-of-plane thickness is 1, and the computation of the in-plane
+residuals is identical to that for the 3D case.
+
+The only exception to this is the plane stress case with finite deformation, because
+the out-of-plane thickness change can be significant, and in general is not spatially
+uniform, so local thickness changes must be accounted for. In this case, the standard
+residual is multiplied by the modified thickness, $t$, which is computed from the logarithmic
+out of plane strain $\varepsilon_{oop}$ as:
+\begin{equation}
+t = e^{\varepsilon_{oop}}
+\end{equation}
+This correction is made for 2D planar models when the deformed mesh is used by setting
+`use_displaced_mesh = true` and `out_of_plane_strain` is specified.
+
+[TensorMechanics Master Action](/Modules/TensorMechanics/Master/index.md). Use of the tensor
```

Comment:
This looks cut off.

Reply:
Removed that line.  It was accidentally pasted there.